gtda/time_series/embedding.py

```diff
@@ -513,7 +509,211 @@ def resample(self, y, X=None):
         check_is_fitted(self)
         yr = column_or_1d(y)
 
-        yr = np.flip(yr)
-        final_index = -self.time_delay_ * (self.dimension_ - 1)
-        yr = np.flip(yr[:final_index:self.stride])
+        final_index = self.time_delay_ * (self.dimension_ - 1)
+        yr = yr[:final_index - 1:-self.stride][::-1]
         return yr
+
+
+@adapt_fit_transform_docs
+class MultiTakensEmbedding(BaseEstimator, TransformerMixin):
+    """Point clouds from collections of time series via independent Takens
+    embeddings.
+
+    On a 1D array representing a single univariate time series, the Takens
+    embedding algorithm is the one described in :class:`TakensEmbedding` and
+    yields a 2D array representing a point cloud in Euclidean space. This
```

Comment:
Maybe change the order of both sentences.

Reply:
I agree, this would improve it.

Reply:
Done.